airbyte-bootloader/src/test/java/io/airbyte/bootloader/SecretMigratorTest.java

```diff
@@ -43,11 +44,14 @@ public class SecretMigratorTest {
   @Mock
   private SecretPersistence secretPersistence;
 
+  @Mock
+  private JobPersistence jobPersistence;
```

Comment:
Should probably add unit tests here for the different cases of `jobPersistence.isSecretMigrated()`

Reply:
Will do.

Reply:
@lmossman I have added more tests in the `BootloaderAppTest` for that.

Reply:
👍 Looks good